staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/envelope.go

```diff
@@ -292,10 +300,23 @@ func validateAnnotations(annotations map[string][]byte) error {
 // 2. The size of keyID is less than 1 kB.
 func ValidateKeyID(keyID string) error {
 	if len(keyID) == 0 {
-		return fmt.Errorf("keyID is empty")
+		return fmt.Errorf(errKeyIDEmpty)
 	}
-	if len(keyID) > keyIDMaxSize {
-		return fmt.Errorf("keyID is %d bytes, which exceeds the max size of %d", len(keyID), keyIDMaxSize)
+	if len(keyID) > KeyIDMaxSize {
+		return fmt.Errorf(errKeyIDTooLong)
```

Comment:
I would leave the `fmt.Errorf` statements unchanged and just return the associated error code along with the error.